group.py

```diff
@@ -2,4 +2,14 @@
 
 # Your code to go here...
 
-my_group =
+my_group =[{ "name": "Luhan", "age": 22, "job": "Student"},   
+        {  "name": "Wangjiayi", "age": 22, "job": "Student"},
+         {"name": "Wangxinping", "age": 22, "job": "Student"}]  
```

Comment:
The information in the data structure looks incomplete :(

See the README for the entire dataset.

Reply:
new commit has made and update the group info